services/QuillLMS/spec/queries/scorebook/query_spec.rb

```diff
@@ -35,6 +35,13 @@
     expect(results.map{|res| res["id"]}).to include(activity_session2.id)
   end
 
+  it 'returns activities even if the underlying activity has been archived' do
+    activity2.update(flags: [Activity::ARCHIVED])
+
+    results = Scorebook::Query.run(classroom.id)
+    expect(results.map{|res| res["id"]}).to include(activity_session2.id)
```

Comment:
I think you can use pluck here:
`expect(results.pluck('id')).to include(activity_session2.id)`